app/connect/page.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Card from '@/components/card/Card';
+import churchBackground from '@/public/church.jpg';
+import test from "@/public/potluck.png"
+
+
+export default function Connect() {
+  return (
+    <div>
+        <div className="justify-center items-center bg-gray-100 h-screen">
+            {/* Image Container */}
+            <div
+                className="flex justify-center items-center w-full h-full bg-cover bg-center bg-opacity-50"
+                style={{ backgroundImage: `url(${churchBackground.src})` }}>
+
+                <h1 className="text-white text-4xl font-bold">Connect With Us</h1>
```

Comment:
looks good but white text is hard to read on this image but it doesnt matter ig (also the navbar is hard to see)